cohortextractor/measure.py

```diff
@@ -0,0 +1,236 @@
+import typing
```

Comment:
This file is mostly just copied over from V1; the `MeasuresManager` is new, but basically does what V1 does outside of `measures.py` to load an input dataframe and calculate each measure from it